docs/concepts/glossary.md

```diff
@@ -97,6 +136,10 @@ Did you mean [IPLD Data Model](https://ipld.io/glossary/#data-model)?
 
 The Datastore is the on-disk storage system used by an IPFS node. Configuration parameters control the location, size, construction, and operation of the datastore. [More about Datastore](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#datastore)
 
+### DCUtR
```

Comment:
How do you even pronounce this? 😅

Reply:
deekatar

Reply:
this is too good to be lost in github comments –  can add this to glossary using https://en.wikipedia.org/wiki/International_Phonetic_Alphabet ? :trollface: